RELEASE.md

```diff
@@ -65,7 +65,7 @@ You will need to authorize your machine to do npm publishing using `npm adduser`
       - [ ] 4b. Run `bower update`. 
       - [ ] 4c. Commit
       - [ ] 4d. Push to `upstream`.
-- [ ] 6. Run `node update-dependencies.js`
+- [ ] 6. Run `node update-dependencies.js [fuelux release #] [lightning theme release #]`
```

Comment:
This isn't user friendly. Either we need to include a flag pattern `node app --flag value --flag2 value2` or break this into prompts in the script.